code/datums/uplink_item.dm

```diff
@@ -12,10 +12,16 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 			var/datum/uplink_item/I = new path
 			if(!I.item)
 				continue
-			if(I.gamemodes.len && SSticker && !(SSticker.mode.type in I.gamemodes))
-				continue
-			if(I.excludefrom.len && SSticker && (SSticker.mode.type in I.excludefrom))
-				continue
+			if(I.include_modes.len)
+				if(!gamemode && SSticker.mode && !(SSticker.mode.type in I.include_modes))
```

Comment:
Before they tested SSticker if it was not null. I doubt it'll ever be null here but something to keep in mind. Same for the if below